uber/tests/models/test_merch.py

```diff
@@ -164,24 +165,41 @@ def test_extra_merch(self):
         assert 'foo' == Attendee(extra_merch='foo').merch
 
     def test_normal_kickins(self, monkeypatch):
-        monkeypatch.setattr(Attendee, 'donation_swag', ['some', 'stuff'])
+        monkeypatch.setattr(Attendee, 'merch_items', ['some', 'stuff'])
+        assert 'some and stuff' == Attendee().merch
+
+    def test_nested_kickins(self, monkeypatch):
+        monkeypatch.setattr(Attendee, 'merch_items', ['some', ['mildly', 'nested'], 'stuff'])
         assert 'some and stuff' == Attendee().merch
 
     def test_comma_and(self, monkeypatch):
-        monkeypatch.setattr(Attendee, 'donation_swag', ['some', 'stuff'])
-        assert 'some, stuff, and more' == Attendee(extra_merch='more').merch
+        monkeypatch.setattr(Attendee, 'merch_items', ['some', 'stuff', 'more'])
+        assert 'some, stuff, and more' == Attendee().merch
+
+    def test_extra_merch(self):
+        assert 'more' == Attendee(extra_merch='more').merch
 
     def test_info_packet(self):
-        assert 'Staffer Info Packet' == Attendee(staffing=True).merch
+        assert '' == Attendee(staffing=True).merch
+        assert 'Staffer Info Packet' == Attendee(staffing=True).staff_merch
 
     def test_staff_shirts(self, gets_staff_shirt):
-        assert '3 Staff Shirts' == Attendee().merch
+        assert 'tshirt' in Attendee().merch
+        assert '2 Staff Shirts' in Attendee().staff_merch
+
+        a = Attendee(second_shirt=c.TWO_STAFF_SHIRTS)
+        assert '' == a.merch
+        assert '3 Staff Shirts' in a.staff_merch
 
     def test_volunteer(self, volunteer_event_shirt_eligible):
-        assert 'RedShirt' in Attendee().merch and 'will be reported' in Attendee().merch
+        assert 'tshirt' in Attendee().merch and 'will be reported' in Attendee().merch
+
+    def test_paid_volunteer(self, paid_for_a_shirt, volunteer_event_shirt_eligible):
+        assert 'RedShirt' in Attendee(amount_extra=c.SHIRT_LEVEL).merch
+        assert '2nd tshirt' in Attendee(amount_extra=c.SHIRT_LEVEL).merch
 
     def test_volunteer_worked(self, volunteer_event_shirt_eligible, volunteer_event_shirt_earned):
-        assert 'RedShirt' == Attendee().merch
+        assert 'tshirt' in Attendee().merch
```

Comment:
I made one additional change to this implementation which is now reflected in our tests.  Previously we'd use whatever the textual description of the tshirt level was.  However, I noticed that in production that level is actually "Tshirt Pack".  So as things were implemented, we would have told the merch booth to give an entire tshirt pack to volunteers instead of just the shirt, which I don't believe is what we want.  (At least, that's never how it worked in the past back when kicking in at the tshirt level included a tshirt / button / stickers / ribbon.)